lib/src/kbasesearchengine/main/IndexerWorker.java

```diff
@@ -476,10 +476,15 @@ private void indexObject(
             throws IndexingException, InterruptedException, RetriableIndexingException {
         long t1 = System.currentTimeMillis();
         final File tempFile;
+        File dir = getTempSubDir(guid.getStorageCode());
```

Comment:
Shouldn't this be inside the try block so SecurityExceptions from mkdirs() are caught?